persistent-mongoDB/Database/Persist/MongoDB.hs

```diff
@@ -605,6 +605,8 @@ instance PersistUniqueRead DB.MongoContext where
         t = entityDef $ Just rec
         rec = dummyFromUnique uniq
 
+    existsBy uniq = isJust <$> getBy uniq
```

Comment:
I'm not very familiar with MongoDB and saw in the readme that the persistent-mongoDB is "on hiatus", so here's a valid impl that doesn't have any performance benefits.